cloudcix_primitives/bridgemain.py

```diff
@@ -0,0 +1,436 @@
+
+            ret = comms_lsh(payloads['delete_bridge'])
+            if ret["channel_code"] != CHANNEL_SUCCESS:
+                return False, fmt.channel_error(ret, f'{prefix + 6}: {messages[prefix + 6]}'), fmt.successful_payloads
+            if ret["payload_code"] != SUCCESS_CODE:
+                return False, fmt.payload_error(ret, f'{prefix + 7}: {messages[prefix + 7]}'), fmt.successful_payloads
+            fmt.add_successful('delete_files', ret)
+
+        ret = comms_lsh(payloads['reload_services'])
+        if ret["channel_code"] != CHANNEL_SUCCESS:
+            return False, fmt.channel_error(ret, f'{prefix + 8}: {messages[prefix + 8]}'), fmt.successful_payloads
+        if ret["payload_code"] != SUCCESS_CODE:
+            return False, fmt.payload_error(ret, f'{prefix + 9}: {messages[prefix + 9]}'), fmt.successful_payloads
+        fmt.add_successful('reload_services', ret)
+
+        return True, "", fmt.successful_payloads
+
+    status, msg, successful_payloads = run_host('localhost', 3120, {})
+    if status is False:
+        return status, msg
+
+    return True, messages[1100]
+
+
+def read(
+        bridge: str,
+) -> Tuple[bool, Dict[str, Any], List[str]]:
+    """
+    description:
+        Reads the service and the vlan tagged bridge on the host .
+
+    parameters:
+        bridge_name:
+            description: Name of the bridge to be created on the PodNet, eg. BM123
+            type: string
+            required: true
+
+    return:
+        description: |
+            A list with 3 items: (1) a boolean status flag indicating if the
+            read was successful, (2) a dict containing the data as read from
+            the both machine's current state and (3) the list of debug and or error messages.
+        type: tuple
+        items:
+          read:
+            description: True if all read operations were successful, False otherwise.
+            type: boolean
+          data:
+            type: object
+            description: |
+              file contents retrieved from Host. May be None if nothing
+              could be retrieved.
+            properties:
+              <host>:
+                description: read output data from machine <host>
+                  type: string
+          messages:
+            description: list of errors and debug messages collected before failure occurred
+            type: array
+            items:
+              <message>:
+                description: exact message of the step, either debug, info or error type
+                type: string
+    """
+
+    up_script_path = f'/usr/local/bin/bridgemain_{bridge}_up.sh'
+    down_script_path = f'/usr/local/bin/bridgemain_{bridge}_down.sh'
+    service_file_path = f'/etc/systemd/system/bridgemain_{bridge}.service'
+
+    # Define message
+    messages = {
+        1200: f'Successfully read bridgemain_{bridge}.service on local host.',
+        1201: f'bridgemain_{bridge}.service does not exists on local host',
```

Comment:
Why is this a success message in read? 

For read, if the service for the bridge is not found, that would be an error.

Reply:
okay